.github/workflows/reusable-api-e2e.yml

```diff
@@ -27,6 +27,7 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   check_submodule_token:
+    if: ${{ inputs.ee }}
```

Comment:
Run this check only if the matrix name is `'novu/api-ee'` otherwise, it is not needed.